charts/spire/charts/tornjak-frontend/values.yaml

```diff
@@ -0,0 +1,48 @@
+# Default values for Tornjak UI (Frontend).
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  # registry: gcr.io
+  # repository: spiffe-io/tornjak-fe
+  registry: ghcr.io
+  repository: spiffe/tornjak-fe
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  version: "v1.0.1"
+
+imagePullSecrets: []
+nameOverride: ""
+namespaceOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+#labels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 3000
+
+
+# -- URL of the Tornjak APIs (backend).
+# The default value is used when both deployed in the same, local cluster.
```

Comment:
```suggestion
```

Would leave this comment out. The default value in a helm chart is always used when the user doesn't override it.

Once we do the ingress followup PR this can probably become the url `http://tornjak.example.org` and in an example we can show to update `/etc/hosts`. This way all is aligned with the other domains and ingresses in the chart.